Air_Traffic_Control_System/src/SSR.h

```diff
@@ -9,7 +9,7 @@
 #define SSR_H_
 #include "Aircraft.h"
 
-class SSR {
+class SSR: public Aircraft {
```

Comment:
Why inherit Aircraft from SSR class? They're going to be two independent entities